lewis/examples/example_motor/__init__.py

```diff
@@ -97,11 +97,14 @@ class ExampleMotorStreamInterface(StreamAdapter):
     The motor starts moving immediately when a new target position is set. Once it's moving,
     it has to be stopped to receive a new target, otherwise an error is generated.
     """
+
+    pattern_type = fmt
+
     commands = {
-        Cmd('get_status', r'^S\?$'),
-        Cmd('get_position', r'^P\?$'),
-        Cmd('get_target', r'^T\?$'),
-        Cmd('set_target', r'^T=([-+]?[0-9]*\.?[0-9]+)$', argument_mappings=(float,)),
+        Cmd('get_status', regex(r'^S\?$')),
+        Cmd('get_position', r'^P?$'),
+        Cmd('get_target', r'^T?$'),
```

Comment:
This has been addressed now for the current version of `scanf` and a PR has been opened to escape those additional characters in `scanf_compile`. We need to remember to go back and change it back when a new version of scanf is available.